backend/src/main/java/org/eclipse/tractusx/puris/backend/demandandcapacitynotification/controller/DemandAndCapacityNotificationRequestApiController.java

```diff
@@ -57,17 +60,19 @@ public class DemandAndCapacityNotificationRequestApiController {
     @PostMapping("request")
     public ResponseEntity<DemandAndCapacityNotificationSamm> postDemandAndCapacityNotification(
         @RequestHeader("edc-bpn") String bpnl,
-        @RequestBody String body)
+        @io.swagger.v3.oas.annotations.parameters.RequestBody(content = {@Content(examples = {
```

Comment:
Not sure if it was like this before but currently the swagger api always has the same example also as return type. At least for the valid response it's a null object. for the badRequest I think it's the same. Would make sense from my point to go either way:
- return the entity / body we received for each response code
- make the swagger ui respond values as currently used

Likely i would just go for the first, but feel free to go what you think would be correct (standard currently does not require this).

Reply:
It was like that before. 

The api returns no response body in any of the error cases, just the HTTP status code. And I think that's what I would show on the swagger ui. 

BTW there are several other endpoints where the situation is currently the same (like for example the production-request-api-controller and others).

Reply:
Fair, thanks for fixing here. Raised further issue #515 to also work on the other controllers accordingly.